CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ adheres to [Semantic Versioning][sv].
 
 [sv]: http://semver.org/
 
+## 0.10.0
+
+* Added 0.15.0 template, which chooses a default graphics backend based on the developer's operating system. ([#104])
```

Comment:
This feature was also added to the 0.12.0 template. And it should probably also be added to 0.13.0 and 0.14.0, for consistency's sake.

Reply:
The link is also broken.

Reply:
ah, thanks for checking in. admittedly been rushing -- shall fix these now.